plugins/logs/plugin.go

```diff
@@ -635,6 +636,11 @@ func (p *Plugin) Reconfigure(_ context.Context, config interface{}) {
 	<-done
 }
 
+// Config returns the plugin's current configuration.
+func (p *Plugin) Config() *Config {
+	return &p.config
+}
+
```

Comment:
I guess we could tighten the interface here, something like
```go
func (p *Plugin) NBDCacheEnabled() bool {
}
```
instead? We don't need to expose the entire config... 🤔

Reply:
I think that's doable, and a good idea. I was copying straight from how the `bundle/` plugin handles exposing its config, and that plugin does actually just expose everything, with helper methods to make specific queries simpler. :sweat_smile:

Reply:
Finally rolled this change into the PR. Marking as resolved. :+1: